Formula/apt.rb

```diff
@@ -0,0 +1,114 @@
+class Apt < Formula
+  desc "Advanced Package Tool"
+  homepage "https://wiki.debian.org/apt"
+  url "https://deb.debian.org/debian/pool/main/a/apt/apt_2.3.7.tar.xz"
+  sha256 "c946a126263b3ea51a15216689602367c0da95d6878b76dc891ea335d87bd936"
+  license "GPL-2.0-or-later"
+
```

Comment:
May also want to add a livecheck. This will depend on how to determine `apt` stable versions.

One option would be to just choose newest version in https://deb.debian.org/debian/pool/main/a/apt

Though, it does seem Debian removes old versions, e.g. linuxbrew-core source tar is no longer available: https://deb.debian.org/debian/pool/main/a/apt/apt_1.9.3.tar.xz

Reply:
I added the `livecheck` block, thanks for providing that.